Code/GraphMol/FMCS/Wrap/testFMCS.py

```diff
@@ -8,251 +8,275 @@
 
 class TestCase(unittest.TestCase):
```

Comment:
unfortunately whatever automatic code formatting tool you used has really screwed up the formatting on this file.
Please try and get it back to 4 spaces for indentation like (most of) the rest of the RDKit.

Reply:
Thanks Greg - I did think about doing something like that and then didn't! I agree it is cleaner and more maintainable though..

Reply:
Not sure what has happened there - I tried to fix it just now via the github web-based editting/commit - not with a great deal of success by the looks of things. I will look again tomorrow as my home internet connection is now being extremely flaky too

Reply:
OK, it seems that what happened, after some careful scouring of both the build output and the source code, was that using the github accept suggested change option on FMCS.cpp added in the new lines suggested be @greglandrum but very helpfully, didnt delete the lines marked in the diff as deleted, which they were supposed to replace.  Hopefully now that is sorted.